src/eclingo/solver/generator.py

```diff
@@ -55,30 +60,41 @@ def __call__(self) -> Iterator[Candidate]:
                 fact(SA).
 
             hold(KA) :- kp_hold(KA).
+
+            positive_extra_assumptions(A) :- epistemic_atom(k(A), KA), kp_hold(KA).
+            % negative_extra_assumptions(A) :- epistemic_atom(k(A), KA), kp_not_hold(KA).
```

Comment:
Is this left out in purpose? If so, why still call the #show negative_extra_assumptions(A)?